neon_llm_core/chatbot.py

```diff
@@ -73,84 +78,115 @@ def ask_discusser(self, options: dict, context: dict = None) -> str:
         :param context: message context
         """
         options = {k: v for k, v in options.items() if k != self.service_name}
-        prompt_sentence = self.prompt_id_to_shout.get(context['prompt_id'], '')
+        prompt_id = context.get('prompt_id') if context else None
+        prompt_sentence = None
+        if prompt_id:
+            prompt_sentence = self.prompt_id_to_shout.get(prompt_id)
         LOG.info(f'prompt_sentence={prompt_sentence}, options={options}')
         opinion = self._get_llm_api_opinion(prompt=prompt_sentence,
-                                            options=options).get('opinion', '')
-        return opinion
+                                            options=options)
+        return opinion.opinion if opinion else "I have nothing to say here..."
```

Comment:
let's maybe retrieve "I have nothing to say here..." to the separate constant, as I see its being referenced multiple times in a code

Reply:
Makes sense; static strings are easier to manage/test with variables too